bin/s2nc.c

```diff
@@ -229,6 +232,31 @@ static void setup_s2n_config(struct s2n_config *config, const char *cipher_prefs
     GUARD_EXIT(s2n_config_send_max_fragment_length(config, mfl_code), "Error setting maximum fragment length");
 }
 
+static void send_cstring(struct s2n_connection *conn, int sockfd, const char *data) {
+    unsigned long bytes_remaining = strlen(data);
+    const char *data_ptr = data;
+    s2n_blocked_status blocked;
+    do {
+        s2n_errno = S2N_ERR_T_OK;
+        ssize_t send_len = MIN(bytes_remaining, INT_MAX);
```

Comment:
I was looking at the echo send code, so I thought setting this was correct/better practice. I removed it.

I also thought it might make sense to make this not cstring specific, so that echo can use it also. Let me know if makes it too complex this way and I will put it back.

Reply:
Nope, this doesn't seem too complex. Seems like a pretty straightforward send!